samples/nrf_desktop/src/hw_interface/motion_optical.c

```diff
@@ -641,10 +676,11 @@ static int init(void)
 static void optical_thread_fn(void)
 {
 	s32_t timeout = K_FOREVER;
+	static u16_t cpi = CONFIG_DESKTOP_OPTICAL_DEFAULT_CPI;
```

Comment:
it's better to keep this on stack (keep things private to the thread)

Reply:
I think this static should be removed. It works the same as `timeout` (a private variable) so it should be placed the same way.